app/assets/stylesheets/comments.scss

```diff
@@ -413,7 +413,7 @@ a.header-link {
         color: var(--form-placeholder);
       }
       &.embiggened {
-        height: calc(2vw + 115px);
+        height: calc(2vw + 106px);
```

Comment:
I changed this value to **106px** from **115px** after experimenting with it making sure it does not change. But I couldn't explain why **106px** fixes it and why it was **115px** before (I couldn't find any reference. Is it sum of height of some elements?).